test/docker-compose-redpanda.yml

```diff
@@ -0,0 +1,24 @@
+version: '3.7'
+networks:
+  redpanda_network:
+    driver: bridge
+volumes:
+  redpanda: null
+services:
+  redpanda:
+    image: docker.redpanda.com/redpandadata/redpanda:v23.1.4
```

Comment:
I see the test works, so I assume that redpanda starts so fast we don't need to wait for a health check to succeed? I mean, if we see the need we can add it in the future, just curious.